src/message/MessageList.js

```diff
@@ -17,6 +17,11 @@ const styles = StyleSheet.create({
 export default class MessageList extends React.PureComponent {
   autoScrollToBottom = false;
 
+  constructor(props) {
+    super();
+    this.state = { actionSheetButtons: ['', ''] };
```

Comment:
But here only once the state is changed, so no issues there. 
Even I don't like the static way of using the actionsheets I saw there is an issue related to this in their library github page as well.